apps/win32/metro.config.js

```diff
@@ -5,10 +5,21 @@
  * @format
  */
 // @ts-check
-const { makeMetroConfig } = require('@rnx-kit/metro-config');
+const { exclusionList, makeMetroConfig, resolveUniqueModule } = require('@rnx-kit/metro-config');
 const MetroSymlinksResolver = require('@rnx-kit/metro-resolver-symlinks');
 const { getDefaultConfig } = require('metro-config');
 
+const excludeMixins = [];
+const extraNodeModules = {};
+function ensureUniqueModule(moduleName, excludeList, nodeModules) {
+  const [nmEntry, excludePattern] = resolveUniqueModule(moduleName);
+  excludeMixins.push(excludePattern);
+  extraNodeModules[moduleName] = nmEntry;
+}
+
+// build up the added excludes and extraNodeModules
+['react-native-svg'].forEach((moduleName) => ensureUniqueModule(moduleName));
```

Comment:
Where does the duplicate come from? Can we just enforce this ourselves via a resolution or aligning every dependency on it now?

Reply:
I'll look at this more when I reenable pnpm.